apps/password-reset/src/assets/i18n/cs.json

```diff
@@ -39,9 +39,9 @@
           "TITLE": "Požadovaný uživatel (dle ID nebo externí identity) neexistuje."
         },
         "SESSION_EXPIRATION": {
-          "TITLE": "Session expiration",
-          "DESCRIPTION": "Your session has expired. Please sign in to continue.",
-          "SIGN_IN": "Sign in"
+          "TITLE": "Platnost přihlášení vypršela",
+          "DESCRIPTION": "Byli jste automaticky odhlášen. Pro pokračování se znovu přihlašte.",
```

Comment:
Either `Byl....odhlášen.` or `Byli....odhlášeni`. Also `přihlaste` and not `přihlašte`. Same for the user-profile app.